task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -305,12 +307,15 @@
         if (event && event.detail) {
           if(event.detail === 'projectView') {
             if (this.task.status.project.id) {
-              let projectManager = []
-              this.$projectService.getProject(this.task.status.project.id).then(data => {
-                projectManager = data.managerIdentities.filter( manager =>  manager.username === eXo.env.portal.userName )
-              }).then(() => {
-                this.isProjectView = projectManager && projectManager.length ? true : false;
-              })
+              this.$projectService.getProject(this.task.status.project.id, true).then(data => {
+                this.isManager = data.managerIdentities.some(manager => manager.username === eXo.env.portal.userName);
+                this.isParticipator = this.isManager ? true : data.participatorIdentities.some(participator => participator.username === eXo.env.portal.userName);
```

Comment:
```suggestion
                this.isParticipator = this.isManager || data.participatorIdentities.some(participator => participator.username === eXo.env.portal.userName);
```

Reply:
This was made in order to avoid a second check since a manager is automatically a member.

Reply:
In OR statements, Javascript will automatically short-circuit the second check if the first condition is true . Check this doc https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR

Reply:
Okay thanks Ali

Reply:
Updated